src/server/views/app.handlebars

```diff
@@ -111,6 +110,16 @@
         .catch(console.error);
     }
 
+    const urlParams = new URLSearchParams(window.location.search);
+    if (urlParams.get('load')) {
+      preloadURL(url);
+    } else {
+      const defaultURL = "{{ defaultURL }}";
+      if (defaultURL) {
+        preloadURL(defaultURL);
+      }
```

Comment:
What is this `{{ defaultURL }}` string? Also, the following `if` statement is always resolving to `true`.